allennlp/models/semantic_parsing/wikitables/wikitables_semantic_parser.py

```diff
@@ -136,14 +132,6 @@ def __init__(self,
             self._question_entity_params = None
             self._question_neighbor_params = None
 
-        self._decoder_step = WikiTablesDecoderStep(encoder_output_dim=self._encoder.get_output_dim(),
-                                                   action_embedding_dim=action_embedding_dim,
-                                                   attention_function=attention_function,
-                                                   num_start_types=self._num_start_types,
-                                                   num_entity_types=self._num_entity_types,
-                                                   mixture_feedforward=mixture_feedforward,
-                                                   dropout=dropout)
-
     @overrides
     def forward(self,  # type: ignore
```

Comment:
I think you can just remove this method entirely.  It's already abstract.

Reply:
Pylint complains if I do that.

Reply:
You should add a `# pylint: disable=abstract-method` at the top of the class definition, because this is still an abstract class.

Reply:
Done.